lib/engine/game/g_18_carolinas.rb

```diff
@@ -7,6 +7,57 @@ module Engine
   module Game
     class G18Carolinas < Base
       load_from_json(Config::Game::G18Carolinas::JSON)
+
+      def init_companies(players)
+        super.take(players.size).each do |company|
+        end
+      end
+
+      def setup
+        counter = 0
+        @companies.each do |company|
+          price = company.value
+          company.owner = @players[counter]
+          @players[counter].companies << company
+          @players[counter].spend(price, @bank)
+          @log << "#{@players[counter].name} buys #{company.name} for #{format_currency(price)}"
+          counter += 1
+          @players.rotate!
+          puts 'rotated'
+        end
+
+        @corporations[0, 1, 2, 3].sort_by! { @game.rand }
+        @corporations[4, 5, 6, 7].sort_by! { @game.rand }
+
+        car_pars = 3
```

Comment:
what's a car par, this should be a constant

Reply:
car_pars was a counter, I changed it to each_with_index.